content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst

```diff
@@ -191,7 +191,50 @@ Now, specific removal strategies can be defined on product categories. To do thi
 category to define the removal strategy on. In the :guilabel:`Force Removal Strategy` field, choose
 a removal strategy.
 
-.. image:: removal/product-category-removal.png
```

Comment:
This image file needs to be removed from the media folder if removed from the rst.